docs/content/troubleshooting/troubleshooting-with-app-protect.md

```diff
@@ -83,7 +83,7 @@ kubectl replace appolicy -f your-policy-manifest.yaml --force
 
 ### Check the Availability of APPolicy External References.
 
-NOTE: This method only applies if you're using [external references](https://docs.nginx.com/nginx-app-protect/configuration/#external-references) in NGINX App Protect policies.
+NOTE: This method only applies if you're using [external references](/nginx-app-protect/configuration/#external-references) in NGINX App Protect policies.
```

Comment:
Does this link work? I wasn't able to test in the preview; got a 404.

Reply:
the preview does not build the APP protect docs. It works in prod (there is a similar link already in prod working just fine)